modular_skyrat/modules/teshari/code/_teshari.dm

```diff
@@ -17,17 +17,21 @@
 		TRAIT_ADVANCEDTOOLUSER,
 		TRAIT_CAN_STRIP,
 		TRAIT_LITERATE,
+		TRAIT_NOSOFTCRIT,
+		TRAIT_EASILY_WOUNDED,
 	)
 	default_mutant_bodyparts = list(
 		"tail" = ACC_RANDOM,
 		"ears" = ACC_RANDOM,
 		"legs" = "Normal Legs"
 	)
 	digitigrade_customization = DIGITIGRADE_NEVER
-	disliked_food = GROSS | GRAIN | GORE
-	liked_food = MEAT
+	disliked_food = GROSS | FRIED
+	liked_food = MEAT | GORE | SUGAR
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
 	payday_modifier = 0.75
+	speedmod = -0.2
```

Comment:
If this is a buff probably best not to do that.

Reply:
This is supposed to be a buff, as all species will get buffs to make them feel more flavorful. 
I've also upped the damage they take and their wounding rate and broken bones. Also, 20% sounds like a ton, but they really do not move all that much faster in the game.

Reply:
This is just stupid.

Teshari with this addition _cannot_ be outrun, and, therefore, chased, by anyone, barring security with disablers (_Which they rarely ever even get on maps to begin with_).

Reply:
> This is just stupid.
> 
> Teshari with this addition _cannot_ be outrun, and, therefore, chased, by anyone, barring security with disablers (_Which they rarely ever even get on maps to begin with_).

yes and no, Teshari, with this addition, can be chased. Yes, they are faster than most crew when unhurt, but if they are hurt or have their legs broken, they will be slowed down and easy to catch. Given these changes, it is not only easier to hurt them, it is easier to break ones leg or tear their muscles with just punches. Not to mention, if you are chasing one, due to the way that stations are laid out, often they will get to the door first and have to wait for it to open, this gives you enough time often to catch up quite easily. 

TLDR; Hurt teshari to slow them down if you need to, throw a chair at them, punch them, keep your mood up so you run faster.

Reply:
> TLDR; Hurt teshari to slow them down if you need to, throw a chair at them, punch them, keep your mood up so you run faster.

So, what you are suggesting is for Sec to have a shotgun pass for Teshari who are running away, which they can't use half the time because the alert isn't high enough for them to carry lethal equipment. Raising alert for a single teshari on the station is also unreasonable.